internal/client/client.go

```diff
@@ -212,6 +212,10 @@ func InitThriftClient(cfg *config.Config) (*ThriftServiceClient, error) {
 			Timeout:   cfg.ClientTimeout,
 		}
 		tTrans, err = thrift.NewTHttpClientWithOptions(endpoint, thrift.THttpClientOptions{Client: httpclient})
+		if err != nil {
+			return nil, err
```

Comment:
I can replace this with a `break` if that's what's preferred here, since it looks like we check for errors immediately after this `switch` block.

Reply:
it is fine like this